update.sh

```diff
@@ -1,11 +1,15 @@
 #!/usr/bin/env bash
+set -x
+set -e
 
-cargo install --force --version 0.13.1 svd2rust &&
-cargo install --force --version 0.99.2 rustfmt-nightly &&
-cargo install --force --version 0.3.0 form &&
+cargo install --force --git https://github.com/wez/svd2rust svd2rust
```

Comment:
Is it possible to at least pin a SHA for this here?

Reply:
oh yeah, will amend.  I had copied the hash earlier but got distracted before pasting it in to the script